pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -13,6 +15,7 @@
 , perl
 , runtimeShell
 , autoconf
+, zlib
```

Comment:
```suggestion
, zlib
```
Would be ideal if alphabetical order was respected.

Reply:
We don't actually need zlib, so it's removed.